Watchman.Engine/Generation/AlarmHelpers.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Amazon.DynamoDBv2;
+using Watchman.AwsResources;
+using Watchman.Configuration;
+using Watchman.Configuration.Generic;
+
+namespace Watchman.Engine.Generation
+{
+    static class AlarmHelpers
+    {
+        public static AlarmValues MergeValueOverrides(string key, Dictionary<string, AlarmValues> serviceThresholds,
```

Comment:
Does making this a static helper not reduce testability inside the generators?

Reply:
Also calling it an alarmhelper now makes it less clear to me what this class actually does. Now that you have moved some of the logic out of here it seems this class is now only responsible for merging alarms together, I feel like the name should reflect this new role, rather than just becoming a generic helper.

Reply:
agree about the naming.
I don't think having the static methods makes it any less testable because they aren't things you would need to fake ,as it's an important part of the class's behaviour and doesn't have any external dependencies.